collections/_blogs/2020-07-22-meshery-accepted-into-cncf-landscape.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Meshery is on the Cloud Native Landscape!
+date:   2020-07-22 11:30:05 -0530
+image: /assets/images/posts/2020-07-22-meshery-on-the-cloud-native-landscape/cncf-meshery.png
+author: Anton Weiss
+permalink: /blog/meshery-on-the-cloud-native-landscape
+---
+
+<div class="center" style="color:#00b39f;position:relative;top:-12px;font-size:1.00em;">This is a story about growing together as a community and building a project that matters.</div>
```

Comment:
At some point, we'll want to have a "subtitle" class both for consistency and for preparing to migrate to ng site. We need to try to not create new styles with every post. There are hundreds of styles in materialize.css already.

Reply:
I'll work on adding that.